modules/ossm-control-plane-deploy.adoc

```diff
@@ -92,9 +89,65 @@ $ oc login https://{HOSTNAME}:6443
 $ oc new-project istio-system
 ----
 
-. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the example found in "Customize the {ProductName} installation". You can customize the values as needed to match your use case.  For production deployments you _must_ change the default Jaeger template.
+. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the following example in "Customize the {ProductName} installation". For production deployments you _must_ change the default Jaeger template.
 
-. Run the following command to deploy the control plane:
+.Example istio-installation.yaml
+[source,yaml]
+----
+apiVersion: maistra.io/v2
```

Comment:
I think this SMCP is wrong. Rob's docs are here: https://github.com/rcernich/istio-operator/blob/apidocgen/docs/crd/maistra.io_ServiceMeshControlPlane_v2.adoc#ControlPlaneSpec and the example generated by the operator is here: 

```
apiVersion: maistra.io/v2
kind: ServiceMeshControlPlane
metadata:
  name: basic
  namespace: openshift-operators
spec:
  version: v2.0
  tracing:
    type: Jaeger
    sampling: 10000
  addons:
    jaeger:
      install:
        storage:
          type: Memory
    kiali:
      enabled: true
    grafana:
      enabled: true
```

Reply:
Most of these fields no longer seem to exist in either.

Reply:
The istio section no longer seems to exist.